scripts/Crawlers/ZenodoCrawler.py

```diff
@@ -186,6 +192,20 @@ def get_all_dataset_description(self):
                         "identifier": "https://doi.org/{}".format(identifier),
                         "identifierSource": "DOI",
                     },
+                    "dates": [
```

Comment:
@emmetaobrien does the dates part sound OK to you for the DATS.json?

Reply:
It looks entirely reasonable to me, and should be straightforward to generate automatically from the GUI too I would expect.